skimage/registration/_phase_cross_correlation.py

```diff
@@ -136,13 +136,13 @@ def _disambiguate_shift(reference_image, moving_image, shift):
     moving_image : numpy array
         The moving image: applying the shift to this image overlays it on the
         reference image. Must be the same shape as the reference image.
-    shift : tuple of float
+    shift : ndarray
```

Comment:
Why update this to `ndarray`? We don't check its type anyway. The only requirement we have is that it's iterable. So something like `sequence of float` would be a more apt description. But I like `tuple of float` best, because defaulting to tuple for simple sequence input types seems like a good default for me.

Reply:
I updated to ndarray because `type(np.array(real_shift_acc)` is an `ndarray` class. I didn't need to do that. I agree with suggestion,`sequence of float` is a more apt description.

Reply:
> Why update this to `ndarray`? We don't check its type anyway.

@lagru ?? As kindly reported by @m-albert in issue #7083 (which this PR is addressing), `ndarray` is the type for the first return value of `ski.registration.phase_cross_correlation` (denoted by `shift`):

* as [documented](https://scikit-image.org/docs/stable/api/skimage.registration.html#skimage.registration.phase_cross_correlation)
https://github.com/scikit-image/scikit-image/blob/2ac3e141e8d2e31aa0ec10afc3a935396b0618fc/skimage/registration/_phase_cross_correlation.py#L248
* and *actually*, whenever `disambiguate=False` (which is the default).

With the current implementation, it is *not* when using `disambiguate=True`, so must either update the docs to reflect this, or fix it (as done in this PR). I support the latter, based on consistency. If you would like to 'relax' `shift` into a tuple, then we would need to update `ski.registration.phase_cross_correlation` and probably `_masked_phase_cross_correlation`, so that it's always a tuple.

I believe it's more natural to keep it an `ndarray`, since it's computed with this kind of operations:
https://github.com/scikit-image/scikit-image/blob/2ac3e141e8d2e31aa0ec10afc3a935396b0618fc/skimage/registration/_phase_cross_correlation.py#L353-L354

Reply:
@mkcor I was confused by @lagru's comment at first but then realised it probably refers to only the `_disambiguate_shift` API, rather than the `_phase_correlation` one which this PR mainly addresses. It's true that in the inner function there's no requirement for `shift` to be a numpy array. It might be less confusing to have the same types for all shifts though.

Reply:
@m-albert thank you for your comment. I agree that there is a question of scope. I believe that the public API should be as consistent as possible, while the inner workings may (should?) follow other considerations such as minimalism, performance, ...

In this case, `phase_cross_correlation` is in the public API, whereas `_disambiguate_shift` is a private function. So, maybe the smarter move (fix) would be to append
```py
shift = np.asarray(shift)
```
to
https://github.com/scikit-image/scikit-image/blob/80442f1cdb55e75ed4b8bb826e0632b4e1aea430/skimage/registration/_phase_cross_correlation.py#L401
?

Reply:
Thanks for taking the time to articulate this @mkcor :) I'd agree that the fix you suggested is the best and probably this is also what @lagru meant.

Reply:
Yes, I think that's what I meant. Though, right now we don't need to append 
```python
shift = np.asarray(shift)
```
because this function already returns an array with
```python
return np.array(real_shift_acc)
```